runtime/compiler/runtime/SymbolValidationManager.hpp

```diff
@@ -794,6 +794,8 @@ class SymbolValidationManager
    void setSymbolOfID(uint16_t id, void *symbol, TR::SymbolType type);
    void defineGuaranteedID(void *symbol, TR::SymbolType type);
 
+   bool isClassWorthRemembering(TR_OpaqueClassBlock *clazz);
```

Comment:
Could we start doxygen'ing the APIs in here - I think it will make it a lot easier for people to find their way in the code...